app/models/application_form.rb

```diff
@@ -101,6 +102,31 @@ def equality_and_diversity_answers_provided?
 
   after_commit :geocode_address_if_required
 
+  def top_ranked_application_choice_status(field)
+    order_sql = Arel.sql("
```

Comment:
Brakeman is flagging this up as a concern. I wanted the ability to pass in one of two possible fields (`:status` and `:status_before_deferral`) - possibly could add some sort of guard cause but is there a better way?